search/search.md

```diff
@@ -303,3 +303,29 @@ Comments are saved in the search history, and are useful for debugging queries a
 ```gravwell
 tag=foo json foo.bar /* a c-style comment that has no impact on the search */ baz | table // another comment!
 ```
+
+## Temporal vs Non-temporal Searches
+
+Gravwell performs searches in two modes: temporal and non-temporal. When you search over time without transforming data, Gravwell will usually perform the search in temporal mode. This mode is often seen with queries that are charted over time, or simple queries using the table renderer. For example, take the following query:
+
+```gravwell
+tag=foo json foo == "bar" | table
+```
+
+This query doesn't transform the data, instead it simply filters entries down to those with a json field named "foo" containing the string "bar". This will result in a temporal search, and when executed, Gravwell will allow you to zoom in to different regions of time over the search results, and will by default sort the data by time, newest entries first.
```

Comment:
I feel like this sentence is really long with a lot of commas.
```suggestion
This query doesn't transform the data, instead it simply filters entries down to those with a json field named "foo" containing the string "bar". This will result in a temporal search. Gravwell will allow you to zoom in to different regions of time over the search results and will by default sort the data by time, newest entries first.
```